src/executables/master_heartbeat.c

```diff
@@ -4393,10 +4394,14 @@ hb_resource_receive_changemode (CSS_CONN_ENTRY * conn)
     {
     case HA_SERVER_STATE_ACTIVE:
       proc->state = HB_PSTATE_REGISTERED_AND_ACTIVE;
+      /* When the ATS is set to active, then the server state is set to ready.
+       * Temporary setting to make understand easier */
```

Comment:
What's it going to be in the future? I mean what is not a temporary setting?

Reply:
This will be discarded in this pull request. It is merely an example illustrating how the server_state is assigned.

Reply:
Aha, in "this" PR before merging. I got it. I guess it's enough and clear to comment.